server/utils/constants.js

```diff
@@ -0,0 +1,19 @@
+const Constants = (module.exports = {})
+
+Constants.Paths = {
+  CHECK_YOUR_ANSWERS: 'check-your-answers',
+  HOME: 'home',
+  IVORY_ADDED: 'ivory-added',
+  IVORY_INTEGRAL: 'ivory-integral',
+  TAKEN_FROM_ELEMENT: 'taken-from-elephant',
+  YES_NO_IDK: 'yes-no-idk'
```

Comment:
Just a minor thing. We don't need a path YES_NO_IDK & the path for HOME is just / so can probably get away with just removing them both?

Reply:
Well spotted, removed.